### DISCUSSION TITLE
Changing /forecast units of measure

### DISCUSSION THREAD

**Discussion by Scott-KC** [weather-gov/api#422] [Q&A] [answered]
I have recently deveoped an iOS weather app, using a lot of the forecast descriptions for each time period. I let the user select their desired unit of measure. However, without digging through the string I would much prefer if the description pull would just give me the same information, but in Celsius for example. I have looked around and tried to change it to 'si' in the URL, but that doesn't seem to work.

**Comment by Scott-KC** [comment, marked as answer]
Nevermind, I figured it out. For anyone else that may have this question after me: ...?units=si

**Reply by roblabs** [reply]
Thanks for sorting this out, Scott-KC.  I would never have found it without your research.  

After digging through the spec, I found this: 

* [https://www.weather.gov/documentation/services-web-api](https://www.weather.gov/documentation/services-web-api)
  * [`[/gridpoints/{wfo}/{x},{y}/forecast`](https://www.weather.gov/documentation/services-web-api#/default/gridpoint_forecast)

---

![](https://user-images.githubusercontent.com/118112/162852366-648a7076-ac97-4368-97f4-6eae6336415c.jpeg)
